cue/ui/gen.cue

```diff
@@ -40,7 +40,7 @@ LineConfig: {
 	lineWidth?:         number
 	lineInterpolation?: LineInterpolation
 	lineStyle?:         LineStyle
-	spanNulls?:         bool
+	spanNulls?:         bool | number
```

Comment:
this is for validate the v8 json, which the value of spanNulls was 3600000

Reply:
Make sure to duplicate this change over in the other cue file from which it was derived (i've just been find/grepping for these)